File: Ejercicios JavaScript/10.Ley Ohm.js
const prompt = require('prompt-sync')();//funcion que permite capturar entradas del usuario para ejecutar en node.js
let Voltaje = Number(prompt('Ingrese el valor del voltaje del circuito: '))
let Resistencia = Number(prompt('Ingrese el valor de la resistencia del circuito: '))
Intensidad = Voltaje/Resistencia //Intesidad entiendase por amperaje
console.log('Al conectar un resistor de: ',Resistencia,'ohm, a una fuente de: ',Voltaje,' V, El voltage circulara una corriente de',Intensidad,'amperios')
//Para ejecutar en terminar: node "C:\Sena\Analisis y desarrollo de software\Ficha 3066478 segundo trimestre\Ejercicios JavaScript\10.Ley Ohm.js"

File: Ejercicios C/10.Ley Ohm.cpp
#include <iostream>
using namespace std;

int main() {
    double Voltaje, Resistencia, Intensidad;  // Usamos tipo double para mayor precisión en los cálculos

    // Solicitar entradas del usuario
    cout << "Ingrese el valor del voltaje del circuito: ";
    cin >> Voltaje;

    cout << "Ingrese el valor de la resistencia del circuito: ";
    cin >> Resistencia;

    // Calcular la intensidad (amperaje)
    Intensidad = Voltaje / Resistencia;

    // Mostrar el resultado
    cout << "Al conectar un resistor de: " << Resistencia << " ohm, a una fuente de: "
         << Voltaje << " V, El voltaje circulara una corriente de " << Intensidad << " amperios." << endl;

    return 0;
}
